lib/ae_mdw/db/mutations/stats_mutation.ex

```diff
@@ -5,25 +5,28 @@ defmodule AeMdw.Db.StatsMutation do
 
   alias AeMdw.Blocks
   alias AeMdw.Db.Model
+  alias AeMdw.Db.Util
   alias AeMdw.Ets
   alias AeMdw.Mnesia
 
   require Model
 
-  defstruct [:height, :prev_stat, :prev_sum_stat, :token_supply_delta]
+  defstruct [:height, :prev_sum_stat, :token_supply_delta]
 
   @opaque t() :: %__MODULE__{
             height: Blocks.height(),
-            prev_stat: Model.stat(),
             prev_sum_stat: Model.sum_stat(),
             token_supply_delta: integer()
           }
 
-  @spec new(Blocks.height(), Model.stat(), Model.sum_stat(), integer()) :: t()
-  def new(height, prev_stat, prev_sum_stat, token_supply_delta) do
```

Comment:
My reasoning behind not doing **any** sort of computation inside the mutation was to be able in the future to run in-memory mutations without having to mock the core or any other external function. IMO, we should try to reach a state where mutations only perform mnesia writes

Reply:
Good goal. Here it cannot be done because it needs the mutations to be executed before getting the results from the cache.

Reply:
But with microblocks transaction it will be (:

Reply:
Sounds good!